src/components/video-editor/VideoEditor.tsx

```
@@ -352,6 +353,20 @@ function getErrorMessage(error: unknown): string {
 
+function getSubtitleExportFileName(
+	format: SubtitleExportFormat,
+	projectPath: string | null,
+	sourcePath: string | null,
+) {
+	const sourceName =
+		(projectPath ?? sourcePath)
+			?.split(/[\\/]/)
+			.pop()
+			?.replace(/\.[^.]+$/, "") ?? "captions";
+	const safeBaseName = sourceName.replace(/[\x00-\x1f<>:"\/\\|?*]+/g, "-").trim();
```

Biome (2.4.15)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)